### PR TITLE
fix: default email used instead of throwing error on timetable submission

### PR DESCRIPTION
**Pull request by Yakitrak**
Ticket: https://pins-ds.atlassian.net/browse/ASB-2327

Changes missing email to use `config.pinsContactDetails.enquiriesEmailAddress` instead of throwing error on exam timetable submission.

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ services:
       FEATURE_GET_UPDATES: 'true'
       FEATURE_ALLOW_HAVE_YOUR_SAY: 'true'
       FEATURE_PROJECT_INFORMATION: 'true'
-      PROJECT_MIGRATION_CASE_REFERENCES: 'EN010009, EN010011, EN010102, EN010085, EN010116, EN013007, EN013009, EN013012, EN013013, EN013014, EN033005, EN043005, EN043006, EN053001, TR010001, TR023018 ,TR023022, TR023024, TR033002, tr033005, TR040009, BC0110001, BC0110002'
+      PROJECT_MIGRATION_CASE_REFERENCES: 'EN010009, EN010011, EN010102, EN010085, EN010116, EN013007, EN013009, EN013012, EN013013, EN013014, EN033005, EN043005, EN043006, EN053001, TR010001, TR023018 ,TR023022, TR023024, TR033002, tr033005, TR040009, BC0110001, BC0110002, BC0110001,BC0110003,BC0110004,BC0110005'
```
Comment (Yakitrak) [inline docker-compose.yml:104]: Added this to help test locally, and thought it would be useful to keep. Happy to remove if that's better.

**Review by christopher-semmens — APPROVED** [review]

**Pull request merged**